src/createUiDefinition.json

```diff
@@ -70,8 +70,10 @@
                         },
                         "toolTip": "Initial password for admin user.",
                         "constraints": {
-                            "required": true
-                        },
+                            "required": true,
+                            "regex": "^(?!(?:adm|admin|audio|backup|bin|cdrom|crontab|daemon|dialout|password|disk|fax|floppy|fuse|games|gnats|irc|kmem|landscape|libuuid|list|lp|mail|man|messagebus|mlocate|netdev|news|nobody|nogroup|operator|plugdev|proxy|root|sasl|shadow|src|ssh|sshd|staff|sudo|sync|sys|syslog|tape|tty|users|utmp|uucp|video|voice|whoopsie|www\\-data)$)\\w+$",
```

Comment:
What do you think about making length the only constraint for the InfluxDB password?

Reply:
Good idea, the rest was over-kill: now se to "at least 7 characters with one alpha-numeric character."